.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+### Ticket
+🎫 [TICKET_ID] <!-- Ticket ID, if there's any put it between brackets -->
+
+### Description
+<!-- Please describe what you have changed or added -->
+<!-- What types of changes does your code introduce? -->
+<!-- Bug fix (non-breaking change which fixes an issue) -->
+<!-- New feature (non-breaking change which adds functionality) -->
+<!-- Include any important information for reviewers -->
+<!-- Etc, etc, etc -->
+
+### Artifacts <!-- if applicable-->
+<!-- 🎥 screencast or 📷 screenshots -->
+
+### Checklist:
+- [ ] I've included a readme entry. <!-- Confirm that it includes the ticket ID -->
+- [ ] My code is tested. <!-- Check that tests are passing and DO NOT merge if they're failing. -->
+- [ ] My code has proper inline documentation.
```

Comment:
We could link this 'proper inline documentation' text to the TEC coding standards page for that